pre_submit.csh

```diff
@@ -201,8 +204,17 @@ sort cam.input_data_list | uniq > cam.input_data_list.sorted
 sort clm.input_data_list | uniq > clm.input_data_list.sorted
 cd -
 
+@ new_month = $data_month + 1
+echo "Now issue a pull request to DART_CASES:"
+echo "   % git status -uno > run_req_${data_year}_${new_month}.csh"
+echo "   Edit run_ to make it `git add` all of the modified files"
+echo "   % csh run_req_YYYY-MM.csh"
```

Comment:
If this is now part of the normal process, run_req_YYYY-MM.csh to the git repo.

Reply:
if all the lines in the output need git add, you could run:

git status -uno | sed -e 's/^/git add /' > run_xxx

and avoid mindless editing.

Reply:
I meant to replace with ${data_year}_${new_month}.
I'll fix that and push again.

Reply:
run_req_YYYY-MM.csh  is (in my mind) a disposable file, now that I think about it.
No need to add it to the git repo.

Nancy -- visually inspecting the contents of the file is a good idea - and if it takes me a second to do a global search/replace inside the file I need to look at anyway - it's not enough to worry about. I think it's the same amount of keystrokes if you do it with sed on the command line or vi inside the file.